 Fix NPE in updateEmptyShadeView() upon screen rotation

FATAL EXCEPTION: main
Process: com.android.systemui, PID: 19252
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.android.systemui.statusbar.EmptyShadeView.willBeGone()' on a null object reference
  at com.android.systemui.statusbar.stack.NotificationStackScrollLayout.updateEmptyShadeView(NotificationStackScrollLayout.java:2291)
  at com.android.systemui.statusbar.phone.NotificationPanelView.updateEmptyShadeView(NotificationPanelView.java:2058)
  at com.android.systemui.statusbar.phone.NotificationPanelView.updateQsState(NotificationPanelView.java:1181)
  at com.android.systemui.statusbar.phone.NotificationPanelView.setQsScrimEnabled(NotificationPanelView.java:2065)
  at com.android.systemui.statusbar.phone.PhoneStatusBar$11.onVerticalChanged(PhoneStatusBar.java:1079)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java b/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java
index 3687512..3660f62 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/stack/NotificationStackScrollLayout.java
@@ -3473,6 +3473,7 @@ public void setEmptyShadeView(EmptyShadeView emptyShadeView) {
     }
 
     public void updateEmptyShadeView(boolean visible) {
+        if (mEmptyShadeView == null) return;
         int oldVisibility = mEmptyShadeView.willBeGone() ? GONE : mEmptyShadeView.getVisibility();
         int newVisibility = visible ? VISIBLE : GONE;
         if (oldVisibility != newVisibility) {
